.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,7 @@
+# Contributing
```

Comment:
Tucking `CONTRIBUTING.md` into the `.github/` directory reduces bulk, noise in the project root directory. It's a way to navigate the tradeoffs in starting to have a `CONTRIBUTING.md` without yet having fully written one.

GitHub will use `CONTRIBUTING.md` in either location. If GitHub finds a `CONTRIBUTING.md` it will draw attention to it in the Pull Request authoring UI.